apps/museums/models/submission.js

```diff
@@ -2,7 +2,7 @@
 
 const _ = require('lodash');
 
-module.exports = data => {
+module.exports = (data, token) => {
```

Comment:
Any `apps/*/models/submission.js` files in all 4 journeys are essentially a function that builds the iCW payload for that particular journey. Now I'm passing the bearer token to each of them as a second parameter we can update all doc urls here instead meaning we can generate the token at the very last minute.